controllers/ratelimitpolicy_limits.go

```diff
@@ -101,112 +94,32 @@ func GetLimitador(ctx context.Context, k8sclient client.Client, rlp *kuadrantv1b
 	return limitador, nil
 }
 
-func (r *RateLimitPolicyReconciler) buildRateLimitIndex(ctx context.Context, rlpRefs []client.ObjectKey) (*rlptools.RateLimitIndex, error) {
+func (r *RateLimitPolicyReconciler) getPolicies(ctx context.Context) ([]kuadrantgatewayapi.Policy, error) {
 	logger, _ := logr.FromContext(ctx)
-	logger = logger.WithName("buildRateLimitIndex").WithValues("ratelimitpolicies", rlpRefs)
 
-	t, err := r.generateTopology(ctx)
+	rlpList := &kuadrantv1beta2.RateLimitPolicyList{}
+	err := r.Client().List(ctx, rlpList)
+	logger.V(1).Info("topology: list rate limit policies", "#RLPS", len(rlpList.Items), "err", err)
 	if err != nil {
 		return nil, err
 	}
 
-	rateLimitIndex := rlptools.NewRateLimitIndex()
-
-	for _, rlpKey := range rlpRefs {
-		if _, ok := rateLimitIndex.Get(rlpKey); ok {
-			continue
-		}
```

Comment:
It looks like we were never updating the Limitador CR after the limit definitions changed in a previously reconciled RateLimitPolicy.

Also fix that.

Reply:
Never mind. What I said is incorrect. This was actually to avoid duplication in the config. Either way, it's no longer needed.